src/ZEO/asyncio/marshal.py

```diff
@@ -21,10 +21,9 @@
 
 import logging
 
-from .._compat import Unpickler, Pickler, BytesIO, PY3
+from .._compat import Unpickler, Pickler, BytesIO
```

Comment:
Why do we import `BytesIO` from `_compat`? The `_compat` definition comes from `ZODB._compat` which uses `io.BytesIO` for PY3 and `cStringIO.StringIO` for PY2.

Reply:
Simply because there was not BytesIO import fixes in #195 and I was doing things only based on that. But you are right - I removed py2 compatibility handling for BytesIO and StringIO throughout in https://github.com/zopefoundation/ZEO/pull/218/commits/1f16336c.